__tests__/app.test.js

```diff
@@ -1,8 +1,20 @@
 const endpointsJson = require("../endpoints.json");
 /* Set up your test imports here */
-
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
```

Comment:
With regards to Test DRIVEN development, do we actually need to seed our database here? Food for thought!